ethereum/datastructures/src/main/java/tech/pegasys/teku/datastructures/util/ValidatorsUtil.java

```diff
@@ -91,12 +94,6 @@ public static boolean is_eligible_for_activation(BeaconState state, Validator va
    * @param state - Current BeaconState
    * @param epoch - The epoch under consideration.
    * @return A list of indices representing the active validators for the given epoch.
-   * @see
-   * @deprecated moved to stateful BeaconStateUtil
```

Comment:
Moved these back to `ValidatorsUtil` - thinking this will make migrations easier if methods don't move around too much.